test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/validate.test.js

```diff
@@ -235,6 +235,52 @@ describe('#validate()', () => {
     expect(() => awsCompileApigEvents.validate()).not.to.throw(Error);
   });
 
+  it('should throw when using a CUSTOM authorizer without an authorizer id', () => {
```

Comment:
For a long time now we're only accepting new tests written with using `runServerless` utility and are slowly migrating away from the old approach. Could you please cover it with such tests instead?

Reply:
No problem, sorry I had missed that earlier in the test README.